file_paths.md

```diff
@@ -197,13 +210,25 @@ user or group.
             puppet.conf *                     # :config
             routes.yaml                       # :route_file
             ssl                               # :ssldir
-    -------------------------------------------------------------------------------------------------------               
+    -------------------------------------------------------------------------------------------------------
         var *
             log                               # :logdir                      same
                 puppet.log                    # not enabled by default
             run                               # :rundir                      same
                 agent.pid                     # :pidfile
 
+    C:\ProgramData\PuppetLabs\pxp-agent *
+        etc *
+            pxp-agent.conf                    # pxp-agent configuration file
+            modules *                         # stores configuration files for pxp-agent modules
```

Comment:
I think modules should be created by packaging (hence the `*`), but I don't think that `pxp-module-puppet` is part of packaging, and won't come down until a connection is established with the master, so I believe we want to remove the `*` from `pxp-module-puppet.conf`